DOTNET_Final_Case_BackEnd/Controllers/ProjectController.cs

```diff
@@ -1,30 +1,31 @@
 using AutoMapper;
+using DOTNET_Final_Case_BackEnd.Dal.Repositories;
 using DOTNET_Final_Case_BackEnd.DTOs.ProjectDTO;
 using DOTNET_Final_Case_BackEnd.Interfaces;
+using DOTNET_Final_Case_BackEnd.Models;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
 using System.Net.Mime;
 
 namespace DOTNET_Final_Case_BackEnd.Controllers
 {
     [ApiController]
-    [Route("[controller]")]
+    [Route("api/projects")]
     [Produces(MediaTypeNames.Application.Json)]
     [Consumes(MediaTypeNames.Application.Json)]
     public class ProjectController : ControllerBase
     {
-        // Private field
+        private readonly ProjectsDbContext _context;
```

Comment:
Here you are using projectDbContext _context. In your ProjectRepository you have also  projectDbContext. I think it is better to use 1 and not 2 DbContext, but If you want to use 2 context is ok.